DisplayBook.tsx

```diff
@@ -47,6 +85,16 @@ const styles = StyleSheet.create({
         height: 180,
         margin: 5,
         backgroundColor: 'gray'
+    },
+    title_container: {
```

Comment:
```suggestion
    titleContainer: {
```
utilise plutot la convention camelCase